R/pkg/R/functions.R

```diff
@@ -1641,30 +1641,30 @@ setMethod("tanh",
           })
 
 #' @details
-#' \code{toDegrees}: Converts an angle measured in radians to an approximately equivalent angle
+#' \code{degrees}: Converts an angle measured in radians to an approximately equivalent angle
 #' measured in degrees.
 #'
 #' @rdname column_math_functions
-#' @aliases toDegrees toDegrees,Column-method
-#' @note toDegrees since 1.4.0
-setMethod("toDegrees",
+#' @aliases degrees degrees,Column-method
+#' @note degrees since 2.1.0
```

Comment:
I'm confused about the since annotation here, where was the degrees implementation in 2.1.0? When I look at https://spark.apache.org/docs/latest/api/R/index.html I don't see the `degrees` function just `toDegrees`>=?

Reply:
yes.. (version here is R API specific)